website/docs/r/redis_cache.html.markdown

```diff
@@ -11,6 +11,8 @@ description: |-
 
 Manages a Redis Cache.
 
+-> **Note:** Redis version 4 is being retired and no longer supports creating new instances. Version 4 will be removed in a future release. [Redis Version 4 Retirement](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-retired-features#important-upgrade-timelines)
```

Comment:
FYI: if you remove the language from the URI (as below) then it'll redirect into the users local language (if available)

```suggestion
-> **Note:** Redis version 4 is being retired and no longer supports creating new instances. Version 4 will be removed in a future release. [Redis Version 4 Retirement](https://learn.microsoft.com/azure/azure-cache-for-redis/cache-retired-features#important-upgrade-timelines)
```